src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h

```diff
@@ -135,20 +136,18 @@ namespace OpenMS
     "X:-C5H8O4-H2S"
   };
 
-  static constexpr std::array<const char*, 13> modifications_RNA_UV_6SG_EXTENDED
+  static constexpr std::array<const char*, 11> modifications_RNA_UV_6SG_EXTENDED
   {   
     "X:",
     "X:-H2O",
     "X:-H3PO4",
     "X:-HPO3",
     "X:-H2S",
-    "X:+HPO3-H2S", // P1 artefact?
     "X:-HPO3-H2S", //pRBSID
     
 
     "X:-C5H8O4",
     "X:-C5H8O4-H2O",
-    "X:-C5H8O4-H3PO4",
```

Comment:
this should be removed artefact 
"X:-C5H8O4-H3PO4", => C5H3N5O-1S1 Artefact,
if other want you can keep, it will not negatively influence